e2e/e2eutil/e2eutil.go

```diff
@@ -169,39 +165,3 @@ func (c *Case) ensureNS() {
 		})
 	}()
 }
-
-// SetupRBAC creates and binds a ClusterRole manifest to a given username. Intended to be used
-// by e2e tests that need to limit their own k8s access.
-func SetupRBAC(cs kubernetes.Interface, roleManifestPath, userName string) error {
```

Comment:
Is RBAC not set up in code anymore because it we have "kubernetes_cluster_role_binding_v1" in terraform? Or is it unrelated?

Reply:
Yeah now we use the service account instead of an impersonated identity. We don't care so much about testing with a minimal role since auth is handled differently than we planned in production, so we just use cluster admin now. Omitting details here to avoid leaking details about internal infra :)